renovate.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended"],
+  "dependencyDashboard": false,
+  "lockFileMaintenance": {
+    "automerge": true,
+    "platformAutomerge": true,
+    "enabled": true,
+    "recreateWhen": "always",
+    "rebaseStalePrs": true,
+    "branchTopic": "lock-file-maintenance",
+    "commitMessageAction": "Lock file maintenance",
+    "commitMessageTopic": null,
+    "commitMessageExtra": null,
```

デフォルトは`UTC`っぽいので，タイムゾーンの変更が必要かなと思いました
```suggestion
    "commitMessageExtra": null,
    "timezone": "Asia/Tokyo",
```